project.clj

```diff
@@ -86,6 +91,7 @@
                          [puppetlabs/jdbc-util "0.6.2"]
                          [puppetlabs/typesafe-config "0.1.5"]
                          [puppetlabs/ssl-utils "0.8.3"]
+                         [puppetlabs/jruby-utils "0.6.0"]
```

Comment:
You're right; this can be fixed entirely in Jar-Jar without any exclusions or pins. I've updated the PR to omit this addition.